public class Valid_palindrome {
    public boolean isPalindrome(String s) {
        s=s.replaceAll("[^a-zA-Z0-9]", "").toLowerCase();
        String rev="";
        for(int i=s.length()-1;i>=0;i--){
        rev=rev+s.charAt(i); }
        if(s.equals(rev)){
            return true;
        }   
        return false;
    }
}
